guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<blockquote>What does <strong>"it"</strong> in this sentence refer to? Is it referring to the <strong>street</strong> or to the <strong>animal</strong>? It's a simple question to a human, but not as simple to an algorithm. When the model is processing the word <strong>"it"</strong>, self-attention allow the model to associate <strong>"it"</strong> with <strong>"animal"</strong>. As each word in the input sequence is processed, self-attention looks at other positions in the input sequence for clues that can lead to a better encoding for this word.</blockquote><p>What does <strong>"it"</strong> in this sentence refer to? Is it referring to the <strong>street</strong> or to the <strong>animal</strong>? It's a simple question to a human, but not as simple to an algorithm. When such data is fed into a transformers model<s>, When the model</s> the model preprocesses <s>processing</s> the word <strong>"it" </strong>and the self-attention algorithm <s>allow</s> allows the model to associate <strong>"it"</strong> with <strong>"animal"</strong>. As each word in the input sequence is processed, self-attention looks at other positions in the input sequence for clues that can lead to a better encoding for this word.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Made the necessary changes</p>